lib/queue_classic_plus/update_metrics.rb

```diff
@@ -18,7 +18,7 @@ def self.metrics
       {
         jobs_queued: jobs_queued,
         jobs_scheduled: jobs_scheduled,
-        max_created_at: max_age("created_at"),
+        max_created_at: max_age("created_at", "created_at = scheduled_at"),
```

Comment:
Yup, possibly, that would make sense I think. I don't think we can wait on this as we have false alarms due to that.